### ISSUE TITLE
[Question] Why CoAP message uploaded to V3.3, device's active is allways 'false'?

### ISSUE THREAD

**Issue by jiekechoo** [thingsboard/thingsboard#5096]
**Component**

 * Generic

**Description**
When a CoAP device upload msg to V3.3, the msg showed on UI, but the server-side attribute 'active' allways 'false'. I changed to HTTP protocol, upload msg again, the attribute 'active' is 'true'. Please help me, how to solve this question?

![image](https://user-images.githubusercontent.com/3350211/129715083-bc7e25ea-a71c-4d60-8138-c5ae56994d76.png)

![image](https://user-images.githubusercontent.com/3350211/129715128-ef73626b-6a9a-4420-9080-99da32bff2b0.png)

**Environment**

 * OS:  CentOS 7.9
 * ThingsBoard: 3.3

**Comment by ashvayka** [comment]
Please let us know more about your setup. Is this a monolith or cluster mode? Which queue you use? etc.

**Comment by jiekechoo** [comment]
Monolith mode in CentOS. In-memory queue. 
Even I set transport type to. default or CoAP, the service-side attrubite: active allways false.
1. setup device profile(transport type: CoAP, coap device: default, coap device payload: json, power saving mode: PSM);
2. add device(Credentials: access_token );
3. my device send msg via CoAP to thingsboard server, telemetry received ok;
4. the server attributes, key:active, value:false;

I chagned active's value to true manually, if device send msg again, the active value kept true. If device stop to send msg, active value changed to false in 10 minutes.

It did not happen in V3.1.1, I have not use V3.2.2 yet. I really love the features in V3.3, it's cool.
